### PR TITLE
Implement drop-down SDK language on left menu

### PR DESCRIPTION
**Pull request by rakeshAlgo**
Hi @amarantha-k , 

I remove the SDK languages from the version drop-down and place it on the left menu. By default, it is open while going on any SDK landing page.
Please look into this and provide your feedback.

And for the main menu button, the SDK landing page menu link is not rendered over this page that why when we click on the back menu button it will not go to the SDK landing page.
Please look into in screenshot below.

leftMenu1
leftMenuAfterClickMainMenuBtn

So, main idea is SDK landing page URL is not available on this page as you can see on the screenshot.
So @mojavelinux , can you suggest to me it will be handled by my side or backend.

### CODE REVIEW

**Comment by mojavelinux** [comment]
@rakeshAlgo Can you rebase this so there's only one commit instead of all the commits from your local merge?

File: src/js/01-nav.js [hunk 1]
```diff
@@ -82,8 +82,10 @@
     navMenuControl.addEventListener('click', function () {
       //   navWrap.style.display = 'none'
       for (var i = 0; i < navItems.length; i++) {
+        console.log(navItems[i], 85)
```
Comment (mojavelinux) [inline src/js/01-nav.js]: All debug statements should be removed before submitting a PR.

File: src/partials/nav-version-control.hbs [hunk 2]
```diff
@@ -16,13 +16,13 @@
                                   <li{{#if (eq ./version @root.page.version)}} class="current"{{/if}}><a class="frame-item" href="{{relativize ./url}}">Version {{./displayVersion}}</a></li>
                               {{/each}}
                               </ul>
-                              {{#if (ends-with page.component.name '-sdk')}}
+                              {{!-- {{#if (ends-with page.component.name '-sdk')}}
```
Comment (mojavelinux) [inline src/partials/nav-version-control.hbs]: Please remove old code instead of commenting out when submitting a PR.

**Comment by mojavelinux** [comment]
> SDK landing page URL is not available on this page as you can see on the screenshot.

Are you asking about the URL to make "SDK" a link?

You can use resolvePageURL to resolve the URL of an arbitrary page, such as `resolvePageURL 'home:sdk.adoc'` or whatever the page ID of the landing page is.

**Comment by rakeshAlgo** [comment]
Yes @mojavelinux , I'm asking about the URL to make the SDK link when we click on the main menu button.

File: src/partials/nav-version-control.hbs (hunk 2 above)
Comment (rakeshAlgo) [reply src/partials/nav-version-control.hbs]: Ok, Sure. I"ll remove all commented code and console things as well.

**Comment by amarantha-k** [comment]
@mojavelinux FYI - we discussed this in the call tonight and I clarified the SDK nav behavior. @rakeshAlgo will be updating the behavior of SDK nav and the "< Main Menu" link too.

**Comment by rakeshAlgo** [comment]
Hi @amarantha-k ,
In the new changes, I worked on making the SDK menu with a two-column layout in version dropdown. Please find below the screenshot. 

image

In this PR border is present under the dropdown link, In the next PR it will be removed.

**Review by amarantha-k — APPROVED** [review]
Noted that SDK lang switch UI will continue to be updated and expect more changes in the next PR.

**Comment by mojavelinux** [comment]
I noticed that the UI project is not configured to allow the SDK menu to be tested in the UI preview. Would you like me to set that up to avoid having to bundle the UI and build the real site to test it? If you'd like me to do that task, please let me know what the priority is.

**Comment by amarantha-k** [comment]
@mojavelinux That would be a good enhancement. However, given the current tasks & priorities, I would say this is low on the priority list.

**Pull request merged**